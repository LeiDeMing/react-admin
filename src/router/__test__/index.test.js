import React from 'react';
import { shallow } from 'enzyme';
import RouterWrap from '../index';


describe('Layout', () => {
    describe('router', () => {
        let element;
        beforeEach(() => {
            element = <RouterWrap />
        })

        test('renders as expected', () => {
            const component = shallow(element);
            expect(component).toMatchSnapshot();
        })

        test('routes /user', () => {
            const component = shallow(element)
            expect(component.find('Route[path="/user"]').first().props('component')).not.toBeUndefined();
        })

        test('routes /', () => {
            const component = shallow(element)
            expect(component.find('Route[path="/"]').first().props('component')).not.toBeUndefined();
        })
    })
})