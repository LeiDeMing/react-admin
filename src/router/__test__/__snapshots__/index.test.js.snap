// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout router renders as expected 1`] = `ShallowWrapper {}`;
